manim/mobject/types/image_mobject.py

```diff
@@ -70,7 +80,7 @@ def construct(self):
         "image_mode": "RGBA",
     }
 
-    def __init__(self, filename_or_array, **kwargs):
+    def __init__(self, filename_or_array, scale_to_resolution=1080, **kwargs):
```

Comment:
Suggestion: instead of hard-coding 1080, maybe you can set a module-global like so
```python
from manim.constats import QUALITIES, DEFAULT_QUALITY
DEFAULT_HEIGHT = QUALITIES[DEFAULT_QUALITY]["pixel_height"]
```

and then define this method as 
```python
def __init__(self, filename_or_array, scale_to_resolution=1080, **kwargs):
```

It's not very pretty, but it's more robust.